src/vendor/processors/stripe.py

```diff
@@ -333,22 +332,50 @@ def get_stripe_connect_account(self):
 
         return None
 
+    def get_stripe_base_fee_amount(self, amount):
+        stripe_base_fee = 0
+
+        if 'percentage' in STRIPE_BASE_COMMISSION and STRIPE_BASE_COMMISSION['percentage']:
+            stripe_base_fee = (amount * STRIPE_BASE_COMMISSION['percentage']) / 100
+
+        if 'fixed' in STRIPE_BASE_COMMISSION and STRIPE_BASE_COMMISSION['fixed']:
+            stripe_base_fee += STRIPE_BASE_COMMISSION['fixed']
+
+        return stripe_base_fee
+
     def get_application_fee_percent(self):
         vendor_site_commission = VendorSiteCommissionConfig(self.site)
 
         if vendor_site_commission.instance:
             return vendor_site_commission.get_key_value('commission')
 
         return None
-
-    def get_application_fee_amount(self):
+  
+    def get_application_fee_amount(self, amount):
         vendor_site_commission = VendorSiteCommissionConfig(self.site)
 
         if vendor_site_commission.instance:
-            return self.convert_decimal_to_integer((vendor_site_commission.get_key_value('commission')*self.invoice.total)/100)
+            return (vendor_site_commission.get_key_value('commission') * amount) / 100
 
         return None
 
+    def get_recurring_fee_amount(self, amount):
+        fee = 0
+
+        if 'percentage' in STRIPE_RECURRING_COMMISSION and STRIPE_RECURRING_COMMISSION['percentage']:
+            fee = (amount * STRIPE_BASE_COMMISSION['percentage']) / 100
+
```

Comment:
just double checking, this is supposed to be using STRIPE_BASE_COMMISSION as the basis for its calculations, not STRIPE_RECURRING_COMMISSION, correct?

Reply:
good catch I thought I had changed that error thanks